Feliz/Properties.fs

```diff
@@ -1042,6 +1567,19 @@ type prop =
     /// Defines the first number if other than 1.
     static member inline start (value: string) = Interop.mkAttr "start" value
 
+    /// Defines the standard deviation for the blur operation.
+    static member inline stdDeviation (value: float) = Interop.mkAttr "stdDeviation" value
+    /// Defines the standard deviation for the blur operation.
+    static member inline stdDeviation (value: int) = Interop.mkAttr "stdDeviation" value
+    /// Defines the standard deviation for the blur operation.
+    static member inline stdDeviation (xAxis: float, yAxis: float) = Interop.mkAttr "stdDeviation" (xAxis, yAxis)
```

Comment:
Same as `radius` property, should not output a tuple but instead serialize the values as `string` into the value